tests/clip/test_modeling_tf_clip.py

```diff
@@ -301,6 +301,12 @@ def prepare_config_and_inputs(self):
         input_mask = None
         if self.use_input_mask:
             input_mask = random_attention_mask([self.batch_size, self.seq_length])
+            # make sure the first token has attention mask `1` to ensure that, after combining the causal mask, there
+            # is still at least one token being attended to for each batch.
+            # TODO: Change `random_attention_mask` in PT/TF/Flax common test file, after a discussion with the team.
+            input_mask = tf.concat(
```

Comment:
This is added to make TF CLIP pass.
(as TF's `random_attention_mask` is changed too in this PR)